### PR TITLE
fix: calculator example

### PR DESCRIPTION
**Pull request by mrxiaozhuox**
The Example `calculator` cannot be use, so I have to fixed the bug, now it's can be run to calculate data.

Some details have yet to be refined.

### CODE REVIEW

File: examples/calculator.rs [hunk 1]
```diff
@@ -18,23 +18,67 @@ fn main() {
     });
 }
 
+fn clac_val(val: String) -> f64 {
```
Comment (autarch) [inline examples/calculator.rs:21]: This should be `calc_val`.

Comment (mrxiaozhuox) [reply examples/calculator.rs]: I changed it.

**Comment by mrxiaozhuox** [comment]
all input will display, and if click some button like: `%` `-/+` `=` then the program will calculate this expression and create a new `value-expression`. so the user can know calculate process, not just display a `0`.

> The original demo code didn't work.

**Comment by jkelleyrtp** [comment]
Thank you so much! I can't believe how broken the original was 😳

Can you clean up/remove the comments and move calc_value after the UI code? Just want to make sure it's polished. Once it's cleaned up a bit, just ping me and we can merge it.

**Comment by mrxiaozhuox** [comment]
> Thank you so much! I can't believe how broken the original was 😳
> 
> Can you clean up/remove the comments and move calc_value after the UI code? Just want to make sure it's polished. Once it's cleaned up a bit, just ping me and we can merge it.

OK: I have formatted this code, and move the `calc_val` after UI code.

**Comment by jkelleyrtp** [comment]
Looks great! Thanks for doing this, the calculator works well :)

**Pull request merged**